Config.in

```diff
@@ -54,10 +54,12 @@ menuconfig BR2_THINGINO_SYSTEM_PACKAGES
 	  https://github.com/thingino/firmware
 
 if BR2_THINGINO_SYSTEM_PACKAGES
+source "$BR2_EXTERNAL_THINGINO_PATH/package/esp-microspeech-features/Config.in"
```

Comment:
I wasn't sure which section these should go in but SYSTEM_PACKAGES seemed ok to me